cartridge-cli.lua

```diff
@@ -340,6 +340,7 @@ local function read_file(path)
     while true do
         local val = file:read(1024)
         if val == nil then
+            pcall(function() file:close() end)
```

Comment:
pcall(file.close, file)

Reply:
I think it looks ugly and confuses a little.

Reply:
Ok, for console application - ok.

If it is in context of server app, I think I will disagree with you